packages/datadog-plugin-jest-circus/src/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  name: 'jest-circus',
+  // ** Important: This needs to be the same as the versions for datadog-plugin-jest
+  versions: ['>=24.8.0']
```

Comment:
ℹ️ there is where the event hooks we use were added to `jest-circus`: https://github.com/facebook/jest/releases/tag/v24.8.0

Reply:
yeah so this one is a bit weird. Our `jest` instrumentation requires `jest-circus`. If I don't specify it here it never gets added when doing `yarn services` and I can't import it in the tests: 

https://github.com/DataDog/dd-trace-js/pull/1157/files#diff-fd24cb8095a6c65c05bff556d3bab1e30e9b7d228ead358834dfd3cdfc25b3b3R22-R24

I talked with Bryan about this. It is not pretty but there seems to be no easy way around it. The testing setup is not really prepared for this. 

What do you think?